glmtools/glm.py

```diff
@@ -54,4 +65,16 @@ def t_test(c, X, B, sigma_2, df):
         two-tailed probability value for each t statistic.
     """
     # Your code code here
-    return
+    c_b_cov = c.dot(npl.pinv(X.T.dot(X))).dot(c)
+
+    t = c.T.dot(B) / np.sqrt(sigma_2 * c_b_cov)
```

Comment:
I think your `c` array is 1D, so the transpose has no effect here.  See : https://bic-berkeley.github.io/psych-214-fall-2016/dot_outer.html for why this code will still work.